R/repr_matrix_df.r

```diff
@@ -77,19 +95,13 @@ ellip.limit.arr <- function(
 		rv <- cbind(a[, left, drop = FALSE], ellip.h, a[, right, drop = FALSE], deparse.level = 0)
 	}
 
-	if (rows < nrow(a)) {
-		# If there were no rownames before, as is often true for matrices, assign them.
-		if (is.null(rownames(rv)))
-			rownames(rv) <- c(top, ellip.v, bottom)
-		else
-			rownames(rv)[[ top[[length(top) ]] + 1L]] <- ellip.v
+	if (rows < nrow(a) && (! is.null(rownames(rv)))) {
+		# If there were no rownames before, as is often true for matrices, don't assign them.
```

Comment:
I added a "don't" to the comment to reflect the change.

These conditionals were generating the incorrect indexing in #28.  Rather than fixing them (which would be very doable, just pulling in rownames from `a` rather than `rv`), I decided that it would be better not to assign names where none existed before.